tests/aws/services/transcribe/test_transcribe.py

```diff
@@ -85,19 +85,20 @@ def is_transcription_done():
         snapshot.match("GetError", e_info.value.response)
 
     @pytest.mark.parametrize(
-        "media_file",
+        "media_file,speech",
         [
-            "../../files/en-gb.amr",
-            "../../files/en-gb.flac",
-            "../../files/en-gb.mp3",
-            "../../files/en-gb.mp4",
-            "../../files/en-gb.ogg",
-            "../../files/en-gb.webm",
+            ("../../files/en-gb.amr", "hello my name is"),
+            ("../../files/en-gb.flac", "hello my name is"),
+            ("../../files/en-gb.mp3", "hello my name is"),
+            ("../../files/en-gb.mp4", "hello my name is"),
+            ("../../files/en-gb.ogg", "hello my name is"),
+            ("../../files/en-gb.webm", "hello my name is"),
+            ("../../files/en-us_video.mkv", "one of the most vital"),
```

Comment:
Can we also add a test case for `mp4` video file, as stated in the github issue?

Reply:
Added in da3d1c85b2a7e5e6aec0a39bc5aa58a2921e5349